### PR TITLE
jitcode act

### PR DESCRIPTION
**Pull request by tensor-tang**
- [x] exp with size 8
- [x] sigmoid with size 8
- [x] tanh with size 8
- [x] remove ComputeDeprecated

- This PR will failed at mac, verified.

exp, sigmoid, tanh 相比MKL至少提升2到4倍左右，后续会吧size做的更通用。
```
exp
I1116 02:59:39.394004 99231 jit_kernel_test.cc:189] Vec size 8: refer takes: 0.1378 us, mkl takes: 0.05915 us, tgt takes: 0.0154
sigmoid
I1116 02:59:39.662591 99232 jit_kernel_test.cc:261] Vec size 8: refer takes: 0.13245 us, better(jit exp) takes: 0.08185 us, tgt takes: 0.02105
vtanh
I1116 02:59:40.214254 99233 jit_kernel_test.cc:329] Vec size 8: refer takes: 0.141 us, better(jit exp) takes: 0.039 us, tgt takes: 0.0189
```

### CODE REVIEW

File: paddle/fluid/operators/math/jit_kernel_exp.cc [hunk 1]
```diff
@@ -30,41 +35,239 @@ namespace math {
...
+}
+#endif
+
+REGISTER_JITKERNEL(vexp, VExpKernel);
+REGISTER_JITKERNEL(vsigmoid, VSigmoidKernel);
+REGISTER_JITKERNEL(vtanh, VTanhKernel);
+
+namespace detail {
 
 #define ALIGN32 __attribute__((aligned(32)))
```
Comment (tensor-tang) [inline paddle/fluid/operators/math/jit_kernel_exp.cc:272]: 272到412还没删掉是因为rnn里面用到了，rnn改完就可以删了。

**Comment by tensor-tang** [comment]
python35 遇到一次 ，已经rerun
358 - test_parallel_executor_test_while_train (Failed)

**Comment by tensor-tang** [comment]
CI 遇到一次 443 - test_label_semantic_roles (Failed) 已经rerun

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**